src/App.css

```diff
@@ -77,17 +72,11 @@ nav ul li a {
   color: #fff;
 }
 
-.book-form input,
-select {
+.book-form input {
```

Comment:
- [ ] Kindly ensure that your form design matches the provided zeplin design 👍🏽 . For you to be professional, it is essential we always stick to design as much as possible.

zeplin design 👇🏽 
![image](https://github.com/DebyGrey/bookstore/assets/63473584/e91bd3c1-e327-4f76-95a4-71817aa25e2e)

your design 👇🏽 
![image](https://github.com/DebyGrey/bookstore/assets/63473584/59c3d0d5-a21e-46ae-a5d4-bfbe9c283f3f)

Reply:
Fixed.